bugbot/rules/inactive_reviewer.py

```diff
@@ -243,9 +243,15 @@ def _fetch_revisions(self, ids: list):
 
     def handle_bug(self, bug, data):
         rev_ids = [
-            int(PHAB_FILE_NAME_PAT.findall(attachment["file_name"])[0])
+            # To avoid loading the attachment content (which can be very large),
+            # we extract the revision id from the file name, which is in the
+            # format of "phabricator-D{revision_id}-url.txt".
+            # len("phabricator-D") == 14
```

Comment:
It's 13.

Reply:
Good catch. Thank you! Fixed in 91a129290db90d62d6e8437202563f7ca4dd667f.